tests/Dockerfile

```diff
@@ -1,5 +1,6 @@
-ARG PYTHON_VERSION=3.6
-FROM python:$PYTHON_VERSION-jessie
+ARG PYTHON_VERSION=3.7
```

Comment:
Shouldn't it be `ARG PYTHON_VERSION=3.7-stretch` to keep the same level of specificity in the TAG?
Or that removal was intentional?

Reply:
They're aliases. Removing it allows overriding it with `PYTHON_VERSION=3.6-jessie` although that would now fail because of the `gpg2` change.

Not sure if there was a specific reason to add the `-jessie` suffix; I couldn't find one directly

Reply:
Hmmm.. checking here, It's not exactly the same image:
```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
python              3.7-stretch         71c6ddda7cab        2 days ago          941MB
python              3.7                 42d620af35be        2 days ago          918MB
```
I'm not against changing to `3.7` instead of `3.7-stretch`. 
My point is just that we have to know it's a different image.

Reply:
Did you check the image's digest as well? Was your image cache out of date perhaps?

```
docker manifest inspect python:3.7
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 2217,
         "digest": "sha256:11454cd111137f54f2c91a624ab16b88d6dd31092a6f7fa267dea5e21b9f0a77",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },

docker manifest inspect python:3.7-buster 
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 2217,
         "digest": "sha256:11454cd111137f54f2c91a624ab16b88d6dd31092a6f7fa267dea5e21b9f0a77",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },


docker manifest inspect python:3.7.4
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 2217,
         "digest": "sha256:11454cd111137f54f2c91a624ab16b88d6dd31092a6f7fa267dea5e21b9f0a77",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },

docker manifest inspect python:3.7.4-buster
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 2217,
         "digest": "sha256:11454cd111137f54f2c91a624ab16b88d6dd31092a6f7fa267dea5e21b9f0a77",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
```

Reply:
Just removed all my images and `docker pull python:3.7` and `docker pull python:3.7-stretch`. By the way, I don't see `python:3.7-stretch` in your message.
My inspect on `python:3.7-stretch` results in:
```
$ docker manifest inspect python:3.7-stretch
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 2218,
         "digest": "sha256:4cb0e5e9861fad660bbe72f0ae5a35e982daad0b7106612aa8ee4c9277faa7d6",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
...
```

Reply:
oh! 🤦‍♂ now see you're comparing with `-stretch`, but `3.7` is equivalent to `-buster` (which is current stable)

Reply:
Ahhh... Ok! I didn't know about `buster` (since it was released July 6th, 2019). In my head `stretch` was still the current stable. Sorry for that :man_facepalming: 
In this case, YES! You are right! I drop my charges :sweat_smile: